pkgs/development/libraries/libnbd/default.nix

```diff
@@ -32,8 +34,24 @@ stdenv.mkDerivation rec {
     libxml2
   ];
 
+  configureFlags = []
+    ++ lib.optionals withPython [ "--with-python-installdir=${placeholder "out"}/${python3.sitePackages}" ];
+
   installFlags = [ "bashcompdir=$(out)/share/bash-completion/completions" ];
 
+  postInstall = lib.optionalString withPython ''
+    mkdir -p ${placeholder "out"}/${python3.sitePackages}/nbd-${version}.dist-info
+    cat <<EOT >> ${placeholder "out"}/${python3.sitePackages}/nbd-${version}.dist-info/METADATA
+      Metadata-Version: 2.1
+      Name: nbd
+      Version: ${version}
+    EOT
+  '';
+
+  outputs = [
+    "out"
+  ];
+
```

Comment:
```suggestion
```

Or is this somehow needed?

Reply:
Oops, that was from and earlier draft with multiple outputs ("out" and "python"). It can be safely removed.